CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+# Contribution Guide
+
+We would love for you to contribute to the project and help make it better than it is today.
+As a contributor, here are the guidelines we would like you to follow:
+
+ - [Question or Problem?](#got-a-question)
+ - [Issues and Bugs](#found-a-bug)
+ - [Feature Requests](#missing-a-feature)
+ - [Submissions](#submission-guidelines)
+ - [Development Guidelines](#development)
+ - [Release Procedure](#release)
```

Comment:
The sections Development Guidelines and Release Procedure don't exist

Reply:
Removed them.